app/src/test/java/com/x8bit/bitwarden/data/platform/repository/util/FakeEnvironmentRepository.kt

```diff
@@ -10,6 +10,9 @@ import kotlinx.coroutines.flow.asStateFlow
  * A faked implementation of [EnvironmentRepository] based on in-memory caching.
  */
 class FakeEnvironmentRepository : EnvironmentRepository {
+    private var saveCurrentEnvironmentForEmailCalled = false
```

Comment:
💭  do we need these `var`s could we get rid of them and just have 

```kotlin
override fun saveCurrentEnvironmentForEmail(userEmail: String) = Unit
...
override fun loadEnvironmentForEmail(userEmail: String): Boolean  = true

Reply:
Thanks you are right I added them but ended up not needing the inner state.